cmd/kops/delete_cluster.go

```diff
@@ -42,6 +43,8 @@ func init() {
 	cmd.Flags().BoolVar(&deleteCluster.External, "external", false, "Delete an external cluster")
 
 	cmd.Flags().StringVar(&deleteCluster.Region, "region", "", "region")
+
+	cmd.Flags().StringVarP(&deleteCluster.Manifest, "manifest", "f", "", "Optional .yml cluster manifest")
```

Comment:
I think the way cobra separates concerns is going to force us defining this for every command. Yes it's duplicate code, but it's probably a good thing as each implementation is subject to change.